test/commands/auth/logout.nut.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { execCmd, TestSession } from '@salesforce/cli-plugins-testkit';
+import { expect } from 'chai';
+import { Env } from '@salesforce/kit';
+import { ensureString, getString } from '@salesforce/ts-types';
+
+let testSession: TestSession;
+
+describe('auth:logout NUTs', () => {
+  const env = new Env();
+
+  before('ensure required environment variables exist', () => {
+    ensureString(env.getString('TESTKIT_JWT_KEY'));
+    ensureString(env.getString('TESTKIT_JWT_CLIENT_ID'));
+    ensureString(env.getString('TESTKIT_HUB_INSTANCE'));
+    ensureString(env.getString('TESTKIT_HUB_USERNAME'));
+  });
+
+  beforeEach(() => {
+    testSession = TestSession.create({});
```

Comment:
I _believe_ that we want to avoid creating too many `TestSessions` and just reuse one and reset/restore it with commands instead of creating new ones... But Steve would know for sure. If you're testing a login flow, call logout in the beforeEach, instead of creating a new testSession